sdks/python/apache_beam/runners/direct/executor.py

```diff
@@ -271,6 +272,14 @@ def __init__(self, transform_evaluator_registry, evaluation_context,
     self._transform_evaluator_registry = transform_evaluator_registry
     self._evaluation_context = evaluation_context
     self._input_bundle = input_bundle
+    # For non-empty bundles, store the window of the max EOW.
+    # TODO(mariagh): Move to class _Bundle's inner _StackedWindowedValues
+    # if input_bundle._elements:
```

Comment:
Please remove commented out code.

Reply:
Both parts are needed.
`# For non-empty bundles, store the window of the max EOW.` was part of the initial PR.
`# TODO(mariagh): Move...` was added as a review request.

Reply:
I got it, the if. Done!

Reply:
Done!